authzed/api/v1/acl_service.proto

```diff
@@ -153,42 +180,60 @@ message CheckPermissionResponse {
   }
 
   ZedToken checked_at = 1;
+
+  // permissionship is the result of the check operation. For a permission, this will be
+  // PERMISSIONSHIP_HAS_PERMISSION if the subject has said permission; for a relation, this will
+  // be PERMISSIONSHIP_HAS_PERMISSION if the subject is a member of the relation via the existance
+  // of a relationship.
```

Comment:
I was a little confused at this first because it didn't start by clarifying that there are two cases: permissions and relations.

Here's my stab it:

>Permissionship communicates whether or not the subject has the requested permission or relation.
>
>This value will be `authzed.api.v1.PERMISSIONSHIP_HAS_PERMISSION` if the requested subject is a member of the computed permission set or there exists a relationship with the requested relation.

Reply:
Modified